app/default/routes.py

```diff
@@ -24,6 +30,39 @@ def not_eligible():
     return render_template("not_eligible.html")
 
 
+@default_bp.route("/continue_application", methods=["GET", "POST"])
+def continue_application():
+    """
+    Page to return to an active application
+    Provides a way of returning to an compelte/incomplete
+        application to complete or make changes
+    The user provides an application id and is redirected to
+        the application summary page within the runner
+    """
+    form = FlaskForm()
```

Comment:
Maybe this should be broken out into a form class somewhere, so we can fill it out with validation for the application_id and any other required fields in future. The following line can then use if form.validate_on_submit() to check validation before using in get_application_data

Reply:
good shout 🔥